node/src/command.rs

```diff
@@ -62,22 +88,37 @@ impl SubstrateCli for Cli {
 		load_spec(id)
 	}
 
-	fn native_runtime_version(_: &Box<dyn ChainSpec>) -> &'static RuntimeVersion {
-		&neumann_runtime::VERSION
+	fn native_runtime_version(
+		chain_spec: &Box<dyn sc_service::ChainSpec>,
+	) -> &'static RuntimeVersion {
+		match chain_spec {
+			chain_spec if chain_spec.is_turing() => {
+				#[cfg(feature = "turing-node")]
+				return &service::turing_runtime::VERSION;
+				#[cfg(not(feature = "turing-node"))]
+				panic!("{}", service::TURING_RUNTIME_NOT_AVAILABLE);
+			},
+			_ => {
+				#[cfg(feature = "neumann-node")]
+				return &service::neumann_runtime::VERSION;
+				#[cfg(not(feature = "neumann-node"))]
+				panic!("{}", service::NEUMANN_RUNTIME_NOT_AVAILABLE);
+			},
+		}
 	}
 }
 
 impl SubstrateCli for RelayChainCli {
 	fn impl_name() -> String {
-		"Neumann Collator".into()
+		"OAK Collator".into()
```

Comment:
is this gonna break the upgrade for neumann?

Reply:
I have no idea. It's something I will need to test today

Reply:
So we're gonna stick with everything as `OAK Collator`. Just closing the loop. that lgtm